File: tests/human/x/php/string_prefix_slice.php
<?php
$prefix = "fore";
$s1 = "forest";
_print(substr($s1, 0, strlen($prefix)) == $prefix);
$s2 = "desert";
_print(substr($s2, 0, strlen($prefix)) == $prefix);

function _print(...$args) {
    $parts = [];
    foreach ($args as $a) {
        if (is_array($a) || is_object($a)) {
            $parts[] = json_encode($a);
        } else {
            $parts[] = strval($a);
        }
    }
    echo implode(' ', $parts), PHP_EOL;
}
?>

File: tests/transpiler/x/clj/string_prefix_slice.clj
(ns main)

(require 'clojure.set)

(def prefix "fore")

(def s1 "forest")

(def s2 "desert")

(defn -main []
  (println (= (subs s1 0 (count prefix)) prefix))
  (println (= (subs s2 0 (count prefix)) prefix)))

(-main)
